include/groonga/db.h

```diff
@@ -42,6 +42,24 @@ GRN_API void
 grn_db_touch(grn_ctx *ctx, grn_obj *db);
 GRN_API grn_rc
 grn_db_recover(grn_ctx *ctx, grn_obj *db);
+/**
+ * \brief Unmaps all opened tables and columns in the passed database.
+ *        Resources used by these opened tables and columns are freed.
+ *        Normally, this API isn't useless. Because resources used by opened
```

Comment:
We used the double negative here so it was a little bit hard for me to understand it.
I guess we might want to express the negative meaning here judging from the following sentence.
```suggestion
 *        Normally, this API isn't necessary. Because resources used by opened
```